mitosheet/mitosheet/tests/public/v3/sheet_functions/bool_functions/test_bool.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Mito.
+# Distributed under the terms of the Modoried BSD License.
+
+"""
+Contains tests for the OR function.
+"""
+
+import pytest
+import pandas as pd
+import numpy as np 
+
+from mitosheet.public.v3.sheet_functions.bool_functions import BOOL
+
+# Raw function tests
+
+BOOL_VALID_TESTS = [
+    ([1], True),
+    ([0], False),
+    ([pd.Series([1])], pd.Series([True])),
+    ([pd.Series([0])], pd.Series([False])),
+    ([pd.Series([0, 1, 0, 2])], pd.Series([False, True, False, True])),
+    ([pd.Series([0, 1, 0, np.NaN])], pd.Series([False, True, False, False])),
+    ([pd.Series(['Hi', 'Hello', 'Nah', 'Doh'])], pd.Series([False, False, False, False])),
+    ([pd.Series([np.NaN, 'Hi', 'Hello', 'Nah', 'Doh'])], pd.Series([False, False, False, False, False])),
```

Comment:
Just calling out that usually, we've been testing the none case with the value `None` instead of np.NaN. It seems like it would not matter which we use for testing though

Reply:
It does matter in the case of string columns, namely because we need to make sure that None's actually get converted to NaNs.